templates/Includes/GitDetailedBuildReference.ss

```diff
@@ -3,46 +3,64 @@
 		<% if $BareURL %>$BareURL<% else %>$URL<% end_if %>
 	</a>
 </h3>
+<% if $Environment.CurrentBuild %>
 
-<div class="row">
-	<div class="col-md-6">
-		<ul class="build-details">
-			<% if $Environment.Project.RepositoryInterface.CommitURL %>
-				<li>
-					<% if $Tags %>
-						<% loop $Tags %>
-							<span class="deployment-tag">$Me</span>
-						<% end_loop %>
+	<div class="row">
+		<div class="col-md-6">
+			<a href="#" id="current-build-toggle">Current Build Details <i class="fa fa-caret-down"></i></a>
+			<div class="table-responsive hide current-build-data">
+				<table class="table">
+					<% if $Environment.CurrentBuild.DeploymentStrategy.getChange("Infrastructure version") %>
+						<tr>
+							<th>Platform Infrastructure</th>
+							<td>$Environment.CurrentBuild.DeploymentStrategy.getChange("Infrastructure version").to</td>
+						</tr>
 					<% end_if %>
-					<a href="{$Environment.Project.RepositoryInterface.CommitURL}/{$SHA}" class="git-sha tooltip-hint">
-						$SHA.ShortHash
-					</a>
-					<% if $Message %><span class="commit-message"> - $Message</span><% end_if %>
-				</li>
-			<% else %>
-				<li><span class="git-sha tooltip-hint">
-					<% if $Tags %>
-						<% loop $Tags %>
-							<span class="deployment-tag">$Me</span>
-						<% end_loop %>
+					<tr>
+						<th>Code Version</th>
```

Comment:
Can we make the order consistent with the deployment plan? i.e. code, infrastructure, base?